media/CircleEditor/index.js

```diff
@@ -537,10 +546,19 @@ host.BrowserHost = class {
         vscode.postMessage({command: 'loadmodel', offset: '0'});
     }
 
-    /**
-     * TODO Implement _msgGetType(message)
-     * Get custom operator attributes type
-     */
+    _msgGetType(message) {
+        const data = message.data;
+        const graphs = data._view._model._graphs;
+        const values = data._type;
```

Comment:
For consistency, following seems better :)

```suggestion
        const types = data._type;
```

Reply:
Thank you!